utils/mosh/Makefile

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mosh
+PKG_VERSION:=1.3.2
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Jakub Tymejczyk <jakub@tymejczyk.pl>
+
+PKG_LICENSE:=GPL-3.0+
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://mosh.org/
+PKG_HASH:=da600573dfa827d88ce114e0fed30210689381bbdcff543c931e4d6a2e851216
+
+PKG_USE_MIPS16:=0
```

Comment:
was this proven to be required? or did you copy it from somewhere else? Is it still relevant after the recent gcc update to gcc 7.x?

Reply:
Without this line there's following error:
{standard input}:1805: Error: opcode not supported on this processor: mips32r2 (mips32r2) `sync' 
It's still present with GCC 7.3.0

Reply:
fair enough.  Was hoping that at least some of mips16 fixes in 7.3 would fix it :)  feel extremely welcome to file a bugreport on gcc, but yeah, this is good then :)